operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -1002,6 +1002,42 @@ local utils = import 'mixin-utils/utils.libsonnet';
       { yaxes: $.yaxes('percentunit') }
     ),
 
+  latencyPanelLabelBreakout(metricName, selector, labels=[], labelReplaceArgSets=[{}], multiplier='1e3')::
```

Comment:
This is just a version of the `latencyPanel` in the upstream grafana.libsonnet. - Will try to upstream it if we like it here